readme.md

```diff
@@ -897,6 +897,8 @@ Default: `[]`
 
 Called with an `Error` instance. The error is passed to the hook right before it's thrown. This is especially useful when you want to have more detailed errors.
 
+**Note:** `RequestError` instances have an `ownContext` property. By default, it is `undefined`. You can freely modify it to gather any data you need to catch.
```

Comment:
Might be nice to make it an empty object by default `Object.create(null)`, so people can just attach properties if they want. Also makes the type nicer.

Should we make the property enumerable?

Reply:
> Should we make the property enumerable?

It is enumerable already.

Reply:
Yeah, I'm asking whether or not it should be enumerable?

Reply:
I would leave it enumerable by default. If people don't want this then they can use `Object.defineProperty`.

Reply:
Ok, I agree. But we should at least document that it's enumerable by default.

Reply:
Followed by an example on how to make it not enumerable :)